docs/en/styling-and-theming/basic-usage.md

```diff
@@ -0,0 +1,177 @@
+# Basic Usage
+
+## Styling a widget
+
+For a given application (this will be relevant in later examples when specifying [widget theme keys](./supplemental.md#widget-theme-keys)):
+
+> package.json
+
+```json
+{
+	"name": "my-app"
+}
+```
+
+Defining [per-widget CSS modules](./supplemental.md#structural-widget-styling):
+
+> src/styles/MyWidget.m.css
+
+```css
+.root {
+	font-family: sans-serif;
+}
+```
+
+> src/widgets/MyWidget.tsx
+
+```tsx
+import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+
+import * as css from '../styles/MyWidget.m.css';
+
+export default class MyWidget extends WidgetBase {
+	protected render() {
+		return <div classes={[css.root]}>My Widget</div>;
+	}
+}
+```
+
+## Making a widget themeable
```

Comment:
@sbinge For example, I'm making the widget themeable which is good. However I have no idea which parts of it are important at first glance - I need to deep dive into the supplemental section....

Reply:
Actually, within `basic-usage` I've been assuming a linear reading order, where each example builds upon the previous. Users should hopefully notice the difference between the original `MyWidget` above and the modifications done to it for this example to make it themeable.

But, that intention is not obvious, and breaks down for ad-hoc reading. Unless we have a better way of presenting diffs between examples, it may be worth making each example more standalone as you describe?

Reply:
I think stating up-front that the examples build on one another would help clarify this.

Reply:
Added a note at the top of the file to clarify this.